src/Cthulhu.jl

```diff
@@ -261,14 +262,30 @@ function _descend(term::AbstractTerminal, interp::CthulhuInterpreter, mi::Method
             ci = preprocess_ci!(codeinf, mi, optimize, CONFIG)
             callsites = find_callsites(interp, codeinf, infos, mi, slottypes, optimize; params)
 
-            display_CI && cthulhu_typed(term.out_stream::IO, debuginfo,
-                codeinf, rt, mi;
-                iswarn, hide_type_stable, inline_cost)
+            if display_CI
+                printstyled(IOContext(term.out_stream::IO, :limit=>true), mi.def, '\n'; bold=true)
+                if debuginfo == DInfo.compact
+                    # Eliminate trailing indentation (see first item in bullet list in PR #189)
+                    str = stringify() do io
+                        cthulhu_typed(io, debuginfo, codeinf, rt, mi;
+                                    iswarn, hide_type_stable, inline_cost)
+                    end
+                    rmatch = match(r"\u001B\[90m\u001B\[(\d+)G( *)\u001B\[1G\u001B\[39m\u001B\[90m( *)\u001B\[39m$", str)
+                    if rmatch !== nothing
+                        str = chop(str, tail = 27 + sum(length, rmatch.captures))
```

Comment:
What is the magic number 27?

Reply:
It's the number of characters in that match string aside from what's in the captures. I'll add a comment.

Reply:
Or better yet, I'll just use `rmatch.offset`.